configuration/product_variant/materialsOdersOdoo.csv

```diff
@@ -1,104 +1,191 @@
-init.70ce2591-e116-4cc0-bb45-fa4e7ebd9783,Seringue 60 Cc,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.0761395c-f78e-4ac8-b03a-f7ff4b0dbee8,"Soft Band 4""",uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.fd0d1292-b0cb-4610-91f5-402ab539fd0d,"Soft Band 6""",uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.16c86336-28b2-4191-90fd-d65a90d0e419,Test De Grossesse,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.19a24418-f0e4-4e39-8e08-02c86c67c2a4,Thermometre,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.e63b4641-2a7d-438d-9c29-7f222f896624,Tube De Levin 10Fr,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.2b0e10e2-832e-44a8-9553-51f6109b2ab7,Tube De Levin 12Fr,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.d8b358fa-c5f0-4b64-99fe-e1cfcbcebac6,Tube De Levin 14Fr,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.c362aab3-3c58-4720-b28e-de003b60e56e,Tube De Levin 18Fr,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.99d0993c-0e74-4e21-a348-dce2f09de2ec,Tube De Levin 5Fr,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.7682d767-be40-4d1d-995f-be5dc257f211,Tube De Levin 8Fr,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.0443d9ad-dc3a-4965-85c1-0a56e53383c6,Tube De Levin 16Fr,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.1ff184ff-0cde-43bd-8257-8ea33e8c3f4b,Vicryl 0,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.cf4149fa-4792-4d22-990c-94f4e1a33d10,Vicryl 1,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.9f256f81-8c5a-45ca-b1c9-bad831be8de3,Vicryl 1-0,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.e579a3a9-44da-4e62-a218-1c3881968582,Vicryl 2-0,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.43fbffb2-f477-4ca4-99ec-e1468aef43a1,Vicryl 3-0,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.c73c937a-1e09-41aa-a374-4081e4a1be49,Vicryl 4-0,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.bd9c0730-b581-44af-89fe-76aca91a375d,Vicryl 5-0,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
-init.a49a1c44-d1c2-45f9-a6b6-6cdb9993fbe5,Vicryl 6-0,uom.product_uom_unit,FALSE,TRUE,product.categ_products_materials_orders,Consumable
\ No newline at end of file
+id,name,uom_id/id,purchase_ok,sale_ok,categ_id/id,type,invoice_policy
```

Comment:
@nuhuhmutebi , you've added the `invoice_policy` field and set the value to `Ordered quantities`. Is that intended?
If so, could you tell me what this does?

Reply:
This means you cannot deliver before the items are invoiced. This has been the case for all products (drugs).